### ISSUE TITLE
Add cmd to field send function

### ISSUE THREAD

**Issue by bronvic** [Miol-Mor/battle-game#50]
Like in our examples in wiki `{"cmd": "field", "row_n": 6, "col_n": 8, "hexs" : [...]}`

**Comment by Hairash** [comment]
There is my example of start message:

```
{"cmd":"field","row_n":3,"col_n":4,"field":{"hexes":[
      {"x":0,"y":0,"unit":{"player":1,"hp":10,"attack":[1,2],"speed":1}},
      {"x":0,"y":1},
      {"x":0,"y":2},
      {"x":1,"y":0},
      {"x":1,"y":1,"content":{"type":"wall"}},
      {"x":1,"y":1},
      {"x":2,"y":0,"content":{"type":"wall"}},
      {"x":2,"y":1},
      {"x":2,"y":2},
      {"x":3,"y":0},
      {"x":3,"y":1},
      {"x":3,"y":2,"unit":{"player":2,"hp":5,"attack":[1,4],"speed":2}}
]}}
```

**Comment by Hairash** [comment]
Closed by #56
